source/manual/manage-ruby-dependencies.html.md

```diff
@@ -7,29 +7,54 @@ layout: manual_layout
 parent: "/manual.html"
 ---
 
-We're [obliged to keep our software current][current].
+We're [obliged to keep our software current][current]. To help with this, we use a
+service called [Dependabot][] to perform automated dependency upgrades.
 
-To help with this, we use a service called [Dependabot][] to perform automated dependency upgrades.
+[RFC 126][] describes the custom configuration we have for Dependabot to reduce the
+number of open PRs, number of deployments and effort required to ensure apps always use
+the latest version of libraries that we care about.
 
-## Add Dependabot to a repo
+### Reviewing Dependabot PRs
+
+Dependabot updates occur relatively soon after a new version is published, which means
+there’s a risk of updating to a rogue version. Some updates also contain breaking
+changes, irrespective of when they are published; using Dependabot over “bundle update”
+makes it easier to spot these.
```

Comment:
I don't think the rest of this sentence is very relevant to the rest of the section - we're just setting the context on why reviewing Dependabot PRs properly is important.

```suggestion
changes, irrespective of when they are published.
```